src/qz/communication/UsbIO.java

```diff
@@ -1,21 +1,25 @@
 package qz.communication;
 
-import qz.utils.UsbUtilities;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import qz.ws.SocketConnection;
 
 import javax.usb.*;
 import javax.usb.util.UsbUtil;
 
 public class UsbIO implements DeviceIO {
-
+    private static final Logger log = LogManager.getLogger(UsbIO.class);
     private UsbDevice device;
     private UsbInterface iface;
 
     private boolean streaming;
 
+    private DeviceOptions dOpts;
+    private SocketConnection websocket;
 
-    public UsbIO(DeviceOptions dOpts) throws DeviceException {
-        UsbDevice device = UsbUtilities.findDevice(dOpts.getVendorId().shortValue(), dOpts.getProductId().shortValue());
```

Comment:
It looks like this line was removed when the local param assignments were added, but it should have been kept in alongside those.

Reply:
Whoa, good catch!  Fixing. :)

Reply:
Fixed via 04402bc.